mne_connectivity/viz/tests/test_circle.py

```diff
@@ -0,0 +1,91 @@
+                  'parsopercularis-rh', 'temporalpole-rh',
+                  'rostralanteriorcingulate-rh', 'parstriangularis-rh',
+                  'medialorbitofrontal-rh', 'rostralmiddlefrontal-rh',
+                  'lateralorbitofrontal-rh', 'parsorbitalis-rh',
+                  'frontalpole-rh']
+    label_names = ['bankssts-lh', 'bankssts-rh', 'caudalanteriorcingulate-lh',
+                   'caudalanteriorcingulate-rh', 'caudalmiddlefrontal-lh',
+                   'caudalmiddlefrontal-rh', 'cuneus-lh', 'cuneus-rh',
+                   'entorhinal-lh', 'entorhinal-rh', 'frontalpole-lh',
+                   'frontalpole-rh', 'fusiform-lh', 'fusiform-rh',
+                   'inferiorparietal-lh', 'inferiorparietal-rh',
+                   'inferiortemporal-lh', 'inferiortemporal-rh', 'insula-lh',
+                   'insula-rh', 'isthmuscingulate-lh', 'isthmuscingulate-rh',
+                   'lateraloccipital-lh', 'lateraloccipital-rh',
+                   'lateralorbitofrontal-lh', 'lateralorbitofrontal-rh',
+                   'lingual-lh', 'lingual-rh', 'medialorbitofrontal-lh',
+                   'medialorbitofrontal-rh', 'middletemporal-lh',
+                   'middletemporal-rh', 'paracentral-lh', 'paracentral-rh',
+                   'parahippocampal-lh', 'parahippocampal-rh',
+                   'parsopercularis-lh', 'parsopercularis-rh',
+                   'parsorbitalis-lh', 'parsorbitalis-rh',
+                   'parstriangularis-lh', 'parstriangularis-rh',
+                   'pericalcarine-lh', 'pericalcarine-rh', 'postcentral-lh',
+                   'postcentral-rh', 'posteriorcingulate-lh',
+                   'posteriorcingulate-rh', 'precentral-lh', 'precentral-rh',
+                   'precuneus-lh', 'precuneus-rh',
+                   'rostralanteriorcingulate-lh',
+                   'rostralanteriorcingulate-rh', 'rostralmiddlefrontal-lh',
+                   'rostralmiddlefrontal-rh', 'superiorfrontal-lh',
+                   'superiorfrontal-rh', 'superiorparietal-lh',
+                   'superiorparietal-rh', 'superiortemporal-lh',
+                   'superiortemporal-rh', 'supramarginal-lh',
+                   'supramarginal-rh', 'temporalpole-lh', 'temporalpole-rh',
+                   'transversetemporal-lh', 'transversetemporal-rh']
+
+    group_boundaries = [0, len(label_names) / 2]
+    node_angles = circular_layout(label_names, node_order, start_pos=90,
+                                  group_boundaries=group_boundaries)
+    con = np.random.RandomState(0).randn(68, 68)
+    plot_connectivity_circle(con, label_names, n_lines=300,
+                             node_angles=node_angles, title='test',
+                             )
+
+    pytest.raises(ValueError, circular_layout, label_names, node_order,
+                  group_boundaries=[-1])
+    pytest.raises(ValueError, circular_layout, label_names, node_order,
+                  group_boundaries=[20, 0])
+    plt.close('all')
```

Comment:
`plt.close('all'` should not be necessary if you have the same `autouse` fixture that MNE-Python has that calls `plt.close('all')` automagically

Reply:
I think I copy pasted this directly from MNE-Python, but let me take a look rn.

Reply:
Yeah we still have a lot left in the MNE-Python codebase that can be removed

Reply:
... you appear not to have a `conftest.py` (?), so you might want to make one and copy the `autouse` fixture from MNE-Python from it